ansible/roles/local_files/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: delete the local_files directory (Only way is a delete and re-create until state=empty is implemented)
+  file: path="{{local_files_dir}}/" state=absent
+  when: local_files_archive is defined
+  delegate_to: localhost
```

Comment:
When using `vagrant` I need to add a `become: no` to make the `delegate_to: localhost` work.

Reply:
Works without issues on Tower.